prov/efa/test/efa_unit_test_cq.c

```diff
@@ -974,16 +981,17 @@ void test_efa_cq_read_send_failure(struct efa_resource **state)
 	struct efa_resource *resource = *state;
 	struct efa_unit_test_buff send_buff;
 	struct fi_cq_data_entry cq_entry;
+	struct fi_context2 ctx;
 	fi_addr_t addr;
 	int ret;
 
 	test_efa_cq_read(resource, &addr, IBV_WC_SEND, IBV_WC_GENERAL_ERR,
-			 EFA_IO_COMP_STATUS_LOCAL_ERROR_UNRESP_REMOTE);
+			 EFA_IO_COMP_STATUS_LOCAL_ERROR_UNRESP_REMOTE, (struct efa_context *) &ctx);
 	efa_unit_test_buff_construct(&send_buff, resource, 4096 /* buff_size */);
 
 	assert_int_equal(g_ibv_submitted_wr_id_cnt, 0);
 	ret = fi_send(resource->ep, send_buff.buff, send_buff.size,
-		      fi_mr_desc(send_buff.mr), addr, (void *) 12345);
+		      fi_mr_desc(send_buff.mr), addr, &ctx);
 	assert_int_equal(ret, 0);
```

Comment:
Can you add test cases (asserts) here to check whether your ctx is assigned correctly? similar to other tests